cipher/cbc_hmac.go

```diff
@@ -98,12 +178,67 @@ func (ctx *cbcAEAD) Seal(dst, nonce, plaintext, data []byte) []byte {
 	return ret
 }
 
+// nonce = iv
+// ciphertext = cdata
+// adata = data
+// kdata = doesn't exist yet
 // Open decrypts and authenticates the ciphertext.
+func (ctx *cbcAEAD) OpenEx(dst, nonce, ciphertext, data []byte) ([]byte, error) {
+	if len(ciphertext) < ctx.authtagBytes {
+		return nil, errors.New("square/go-jose: invalid ciphertext (too short)")
+	}
+
+	offset := len(ciphertext) - ctx.authtagBytes - 256 // sha256 digest size
```

Comment:
Thanks, brain wasn't working. You got me. The bytes do seem to be fixed at 256, but it's not a SHA digest.